backend/data/src/main/kotlin/io/tolgee/formats/xliff/util/XliffHtmlEscaper.kt

```diff
@@ -0,0 +1,13 @@
+package io.tolgee.formats.xliff.util
+
+object XliffHtmlEscaper {
```

Comment:
I believe we already have apache commons package dependency, which we can use for XML escaping:
```
import org.apache.commons.text.StringEscapeUtils

fun escapeXmlContent(input: String): String {
    return StringEscapeUtils.escapeXml10(input)
}

fun main() {
    val rawText = """<tag attribute="value">Content & more</tag>"""
    val escapedText = escapeXmlContent(rawText)
    println(escapedText) // Output: &lt;tag attribute=&quot;value&quot;&gt;Content &amp; more&lt;/tag&gt;
}
```

I believe they have it tested well.